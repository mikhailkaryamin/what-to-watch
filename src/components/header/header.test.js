import React from 'react';
import renderer from 'react-test-renderer';

import Header from './header.jsx';

describe(`Header`, () => {
  const BG_IMAGE = `img/aviator.jpg`;

  const wrapper = renderer
    .create(<Header backgroundImage={BG_IMAGE} />);

  test(`should render header`, () => {
    expect(wrapper).toMatchSnapshot();
  });
});
