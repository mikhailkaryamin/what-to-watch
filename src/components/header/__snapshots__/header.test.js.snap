// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render header 1`] = `
Array [
  <div
    className="movie-card__bg"
  >
    <img
      alt="The Grand Budapest Hotel"
      src="img/aviator.jpg"
    />
  </div>,
  <h1
    className="visually-hidden"
  >
    WTW
  </h1>,
  <header
    className="page-header movie-card__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link "
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="user-block"
    >
      <a
        className="user-block__link"
        href="/login"
        onClick={[Function]}
      >
        Sign in
      </a>
    </div>
  </header>,
]
`;
